ethcore/src/spec/spec.rs

```diff
@@ -91,36 +91,36 @@ pub struct CommonParams {
 	pub eip161d_transition: BlockNumber,
 	/// Number of first block where EIP-98 rules begin.
 	pub eip98_transition: BlockNumber,
-	/// Number of first block where EIP-658 rules begin.
+	/// Number of first block where EIP-658 (BYZANTIUM) rules begin.
 	pub eip658_transition: BlockNumber,
 	/// Number of first block where EIP-155 rules begin.
 	pub eip155_transition: BlockNumber,
 	/// Validate block receipts root.
 	pub validate_receipts_transition: BlockNumber,
 	/// Validate transaction chain id.
 	pub validate_chain_id_transition: BlockNumber,
-	/// Number of first block where EIP-140 (Metropolis: REVERT opcode) rules begin.
+	/// Number of first block where EIP-140 (BYZANTIUM: REVERT opcode) rules begin.
 	pub eip140_transition: BlockNumber,
-	/// Number of first block where EIP-210 (Metropolis: BLOCKHASH changes) rules begin.
+	/// Number of first block where EIP-210 (CONSTANTINOPLE: BLOCKHASH changes) rules begin.
 	pub eip210_transition: BlockNumber,
 	/// EIP-210 Blockhash contract address.
 	pub eip210_contract_address: Address,
 	/// EIP-210 Blockhash contract code.
 	pub eip210_contract_code: Bytes,
 	/// Gas allocated for EIP-210 blockhash update.
 	pub eip210_contract_gas: U256,
-	/// Number of first block where EIP-211 (Metropolis: RETURNDATASIZE/RETURNDATACOPY) rules
+	/// Number of first block where EIP-211 (BYZANTIUM: RETURNDATASIZE/RETURNDATACOPY) rules
 	/// begin.
 	pub eip211_transition: BlockNumber,
-	/// Number of first block where EIP-214 rules begin.
+	/// Number of first block where EIP-214 (BYZANTIUM) rules begin.
 	pub eip214_transition: BlockNumber,
-	/// Number of first block where EIP-145 rules begin.
+	/// Number of first block where EIP-145 (CONSTANTINOPLE) rules begin.
 	pub eip145_transition: BlockNumber,
-	/// Number of first block where EIP-1052 rules begin.
+	/// Number of first block where EIP-1052 (CONSTANTINOPLE) rules begin.
 	pub eip1052_transition: BlockNumber,
-	/// Number of first block where EIP-1283 rules begin.
+	/// Number of first block where EIP-1283 (CONSTANTINOPLE) rules begin.
 	pub eip1283_transition: BlockNumber,
-	/// Number of first block where EIP-1283 rules end.
+	/// Number of first block where EIP-1283 rules end. (CONSTANTINOPLE)
```

Comment:
This is actually `(PETERSBURG)`.

Reply:
I think we should revert all those comments because those are unrelated to this PR better to fix another one as I suggested when I reviewed it

Reply:
I agree it's unrelated but I wouldn't be too strict here as it is just docs and the added value is actually something we should welcome; and also makes slightly sense since we add chain configs for 3 other networks. So I'd say, let's keep it.